kod/util/rntmaint.kod

```diff
@@ -169,7 +175,26 @@ messages:
             #parm1=initCost, #parm2=perDayCost, #parm3=keyCopyCost);
 
          return;
-      }    
+      }
+      
+      % Command: Change Lock
+      if StringContain(speech, RoomMaintenance_command_change_lock)
+         OR StringContain(speech, "change lock")
+      {
+         rSayRsc = Send(self, @ChangeLock, #who=what, #iLocation=innRid);
+
+         % On success, rSayRsc will be $, so respond in the innkeeper's local flavor
```

Comment:
The non-existent documentation for ChangeLock doesn't say what its return value is.  Add a document string to it.